apis/placement/v1beta1/stageupdate_types.go

```diff
@@ -396,6 +426,11 @@ const (
 	// - "True": The staged update run is completed successfully.
 	// - "False": The staged update run encountered an error and stopped.
 	StagedUpdateRunConditionSucceeded StagedUpdateRunConditionType = "Succeeded"
+
+	// StagedUpdateRunConditionAbandoned indicates whether the staged update run has been abandoned.
+	// Its condition status can be one of the following:
+	// - "True": The staged update run has been abandoned by user request.
+	StagedUpdateRunConditionAbandoned StagedUpdateRunConditionType = "Abandoned"
```

Comment:
do we need this?

stopped is "progressing == false" and "reason is stopped"

Abandoned is "progressing == false" and "reason is Abandoned"